index.php

```diff
@@ -6,7 +6,7 @@
     get_template_part( 'inc/sub-header' );
         $sticky = get_option( 'sticky_posts' );
         $args   = array(
-             'posts_per_page' => 1,
+            'posts_per_page' => 1,
```

Comment:
I believe this is the query that needs to change its arguments. The page builds its "featured" card in parallel to the main loop, so expanding that area from 1 to 4 should change this value.